edg/parts/JlcCapacitor.py

```diff
@@ -8,24 +8,28 @@
 class JlcCapacitor(TableDeratingCapacitor, CeramicCapacitor, SmdStandardPackageSelector, JlcTableSelector):
   PACKAGE_FOOTPRINT_MAP = {
     # 0201 not in parts table, C_0201_0603Metric
-
+    '402': 'Capacitor_SMD:C_0402_1005Metric',
     '0402': 'Capacitor_SMD:C_0402_1005Metric',
+    '603': 'Capacitor_SMD:C_0603_1608Metric',
     '0603': 'Capacitor_SMD:C_0603_1608Metric',
+    '805': 'Capacitor_SMD:C_0805_2012Metric',
     '0805': 'Capacitor_SMD:C_0805_2012Metric',
     '1206': 'Capacitor_SMD:C_1206_3216Metric',
+    '1210': 'Capacitor_SMD:C_1210_3225Metric',
     '1812': 'Capacitor_SMD:C_1812_4532Metric',
```

Comment:
Support for 1210 makes sense.

I'm less convinced about the need to add parsing for `402`/`603`/`805`, I looked through the pruned table and only `603` shows up for capacitors, in one line item.

In the future, the right answer might be to delegate parsing / name canonicalization to another project, like https://github.com/yaqwsx/jlcparts. The current pruned parts library is pretty outdated, though it still does work, and has the benefit of repeatable unit tests.

Reply:
I think that's Excel interpreting the field as a number and dropping the leading zero in its presentation. In reality most of those entries (with only 3-4 exceptions, I think) have the leading zero.

In my version of Excel there's the option to do data import without interpreting the data types, that should preserve the original formatting.